src/antelope/wallets/ual/utils/metakeep-cache.ts

```diff
@@ -0,0 +1,131 @@
+// utils/metakeep-cache.ts
+
+export interface MetakeepCacheData {
```

Comment:
This is an auxiliary file with useful functions to keep cached all data coming from metakeep, so we don't need to query the same data over and over again each time the user logs in. That makes the process of auto-log much faster.